src/bin/worker.ts

```diff
@@ -0,0 +1,7 @@
+import { expose } from "threads/worker"
```

Comment:
This should got to `src/worker.ts` or `src/workers/worker.ts` like we do in `js-encryptedfs` or `js-polykey`.

Reply:
moved to `src/lib/workers/worker.ts` and moved the workers test into `src/lib/workers/test-workers.ts`